docs: Add example usage of reconciliationPaused feature (#648)
stackabletech/documentation: modules/concepts/pages/operations/cluster_operations.adoc

If not specified, `clusterOperation.reconciliationPaused` and `clusterOperation.stopped` default to `false`.

> IMPORTANT: In case you create a stacklet and set `reconciliationPaused` to `true` during the creation, no resources will be created.

Reviewer: ```suggestion
IMPORTANT: If you create a stacklet and set `reconciliationPaused` to `true` during the creation, no resources will be created.
```

> IMPORTANT: In case you create a stacklet and set `reconciliationPaused` to `true` during the creation, no resources will be created.

Author: I did go with @NickLarsenNZ suggestion, hope that works for you

> This is due to the fact that you basically state "Hey, my-operator, here is a MyCluster, but please ignore it".

Reviewer 2: I would reword it less casually. Will come back with a suggestion soon.

> IMPORTANT: If you create a stacklet and set `reconciliationPaused` to `true` during the creation, no resources will be created.
> This is due to the fact that you basically state "Hey, my-operator, here is a MyCluster, but please ignore it".

Reviewer 2: ```suggestion
[IMPORTANT]
===========
When `reconciliationPaused` is set to `true`, operators will ignore reconciliation events (creations, updated, deletions).

Furthermore, if you create a stacklet where `reconciliationPaused` is set to `true`, no resources will be created.
===========
```

> This is due to the fact that you basically state "Hey my-operator, here is an MyCluster, but please ignore it".

Reviewer: ```suggestion
This is due to the fact that you basically state "Hey, my-operator, here is a MyCluster, but please ignore it".
```

If not specified, `clusterOperation.reconciliationPaused` and `clusterOperation.stopped` default to `false`.

[IMPORTANT]

> ===========
> When `reconciliationPaused` is set to `true`, operators will ignore reconciliation events (creations, updates, deletions).
> 
> Furthermore, if you create a stacklet where `reconciliationPaused` is set to `true`, no resources will be created.
> ===========
> 
> IMPORTANT: When setting `clusterOperation.reconciliationPaused` and `clusterOperation.stopped` to true in the same step, `clusterOperation.reconciliationPaused` will take precedence.
> This means the cluster will stop reconciling immediately and the `stopped` field is ignored.
> To avoid this, the cluster should first be stopped and then paused.

Reviewer 2: My apologies, [blocks need (?) to use _four_ `=`](https://docs.antora.org/antora/latest/asciidoc/examples/#delimited). I thought it just had to match :shrug: 

Also, I forgot to delete the previous text in the suggestion.

```suggestion
====
When `reconciliationPaused` is set to `true`, operators will ignore reconciliation events (creations, updates, deletions).

Furthermore, if you create a stacklet where `reconciliationPaused` is set to `true`, no resources will be created.
====
```

Author: Applied in 688a3e5c
